docker-compose-koillectionPG.yml

```diff
@@ -0,0 +1,46 @@
+#This docker-compose file is provided for dev purposes
+version: '3'
+
+services:
+    #FPM image
+    koillection:
+        container_name: koillection
+        build:
+            dockerfile: Dockerfile.dev
+        restart: always
+        ports:
+            - 81:80
+        environment:
+            - APP_ENV=dev
+            - APP_DEBUG=1
+            - DB_DRIVER=pdo_mysql
+            - DB_NAME=koillection
+            - DB_HOST=mysql
+            - DB_PORT=3306
+            - DB_USER=root
+            - DB_PASSWORD=password
+            - DB_VERSION=8
+            - PHP_TZ=Europe/Warsaw
+            - HTTPS_ENABLED=0
+        depends_on:
+            - mysql
+        volumes:
+            - "./docker/volumes/koillection/public/uploads:/var/www/koillection/public/uploads"
+            - ./:/var/www/koillection
+
+    mysql:
+        container_name: mysql
+        image: mysql:latest
+        environment:
+            - MYSQL_DATABASE=koillection
+            - MYSQL_ROOT_PASSWORD=password
+        volumes:
+            - "./docker/volumes/mysql:/var/lib/mysql"
+
+    phpmyadmin:
+        image: phpmyadmin
+        restart: always
```

Comment:
No need? In case of crash no need to spam logs? But maybe I miss sth.

Please consider.

Reply:
Ok. In case of restart of the server it will restart the container automatically.

Reply:
According to the docker docs:

> Always restart the container if it stops. If it's manually stopped, it's restarted only when Docker daemon restarts or the container itself is manually restarted. 

I agree that it can be useful in db, too - added